api/v2/controllers/logs/logs_controller.py

```diff
@@ -0,0 +1,42 @@
+from flask import make_response, jsonify, request
+from flask_restful import Resource
+from api.v2.utilities.logs_validator import validate_log_data
+from api.v2.models.logs.logs_model import Logs as LogsModel
+from api.v2.models.logs.logs_schema import LogsSchema
+
+import datetime
+
+class Logs(Resource):
+    method_decorators = [validate_log_data]
+    def get(self):
+        '''
+        Function to get all logs from the database
+        @return a list of logs
+        '''
+        try:
+            get_logs = LogsModel.query.all()
+            logs = 'No logs found'
+            if get_logs:
+                logs = LogsSchema(many=True).dump(get_logs)
+                return make_response(jsonify({'logs': logs}), 200)
+        except:
+            return make_response(jsonify({'error': 'server error! please try again later'}), 500)
+
+
+    def post(self):
```

Comment:
Let's remove this endpoint, we do not want logs being updated over a public endpoint. The `save_log` method will just have to be called in the notification functionality.